src/coreComponents/mainInterface/ProblemManager.cpp

```diff
@@ -913,7 +913,7 @@ map< std::tuple< string, string, string, string >, localIndex > ProblemManager::
 
             if( feDiscretization != nullptr )
             {
-              elemRegion.forElementSubRegions< CellElementSubRegion, FaceElementSubRegion >( [&]( auto & subRegion )
+              elemRegion.forElementSubRegions< CellElementSubRegion >( [&]( auto & subRegion )
```

Comment:
```suggestion
              elemRegion.forElementSubRegions< CellElementSubRegion >( [&]( auto & CellElementSubRegion )
```
and you may be able to simplify into
```c++
fe = subRegion.registerWrapper< finiteElement::FiniteElementBase >( discretizationName, std::move( newFE ) ).
               setRestartFlags( dataRepository::RestartFlags::NO_WRITE ).reference();
```
and remove
```c++
using SUBREGION_TYPE = TYPEOFREF( subRegion );
```
?

Reply:
we can but we will likely have to put it back coz this fix should be temporary. I just don't know what elemType is appropriate for the `FaceElements` since they are heterogeneous. And I don't think we want to allocate always the max number of quadrature points. Sounds like a huge waste.

Reply:
> we can but we will likely have to put it back coz this fix should be temporary. I just don't know what elemType is appropriate for the `FaceElements` since they are heterogeneous.

A lot of water will have passed under the bridge when we have a solution for that.
I think that nobody's working on that subject?

> And I don't think we want to allocate always the max number of quadrature points. Sounds like a huge waste.

Not such a huge waste since fractures have little number of elements w.r.t. 3d elements. 🤷

Reply:
> Not such a huge waste since fractures have little number of elements w.r.t. 3d elements. 🤷

I would be really careful about that assumption, see examples coming out of Kinetix https://www.software.slb.com/products/kinetix, it could be a lot of fractures

Reply:
Do you have figures? I can only see pictures, I'd be interested in knowing a little more.
But in that case, we'll have to manage this. Still, I don't think  anyone's working on the subject 🤣 🤷

Reply:
> Do you have figures? I can only see pictures, I'd be interested in knowing a little more. But in that case, we'll have to manage this. Still, I don't think anyone's working on the subject 🤣 🤷

check for example
https://onepetro.org/REE/article/26/03/888/516635/A-Hybrid-Embedded-Discrete-Fracture-Model-and-Dual?searchresult=1